test/integration/TextEditorTest.kt

```diff
@@ -116,24 +125,43 @@ class TextEditorTest : IntegrationTest() {
     fun schemaWriteAndRollback() {
         withTypeDB { typeDB ->
             runBlocking {
+                Service.notification.dismissAll()
+
+                val commitDateAttributeName = "commit-date"
                 copyFolder(source = SampleGitHubData.path, destination = testID)
                 openProject(composeRule, projectDirectory = testID)
                 connectToTypeDB(composeRule, typeDB.address())
                 createDatabase(composeRule, dbName = testID)
 
                 Service.client.session.tryOpen(testID, TypeDBSession.Type.SCHEMA)
-                delayAndRecompose(composeRule, Delays.NETWORK_IO)
+
+                waitUntilTrue(composeRule) {
+                    Service.client.session.type == TypeDBSession.Type.SCHEMA
+                }
 
                 clickText(composeRule, Label.SCHEMA.lowercase())
                 clickText(composeRule, Label.WRITE.lowercase())
 
                 Service.project.current!!.directory.entries.find { it.name == SampleGitHubData.schemaFile }!!
                     .asFile().tryOpen()
 
-                clickIcon(composeRule, Icon.RUN, delayMillis = Delays.NETWORK_IO)
-                clickIcon(composeRule, Icon.ROLLBACK, delayMillis = Delays.NETWORK_IO)
+                clickIcon(composeRule, Icon.RUN)
+
+                waitUntilTrue(composeRule) {
```

Comment:
This is something we're asserting, and hence, should have the word `assert` in it.